### ISSUE TITLE
Ignore closed roads for outdoor profiles

### ISSUE THREAD

**Issue by karussell** [graphhopper/directions-api-java-client#32]
Often bike and foot routing is used for planning and so the current behaviour of excluding roads because they are closed in winter is not perfect if you plan your summer trip. 

Another possibility needs a bit more work but is already in the works: implementing a more generic profile where stuff like this or other preferences can be changed per-request.

**Comment by boldtrn** [comment]
Isn't this an issue of the GHCore/API itself? If the API supports a request parameter for that behavior this should work in the client without any changes.

**Comment by karussell** [comment]
Yes, indeed. But we could also deploy a foot&bike installation where we ignore conditional restrictions as a workaround (although I wouldn't like this too much)

**Comment by boldtrn** [comment]
> But we could also deploy a foot&bike installation where we ignore conditional restrictions

Ah ok, now I get it :).

> although I wouldn't like this too much

No, not the cleanest solution. We could also setup two encoders for these types, for example foot and foot_unrestricted.
